### DISCUSSION TITLE
Limit characters on calculations field?

### DISCUSSION THREAD

**Discussion by carolineboeykens** [verbb/formie#1927] [Q&A]
Is there a limit on the characters of the calculations field?

We think that we are not able to get to the end of this calculation.

![image](https://github.com/verbb/formie/assets/42932673/9fbe640c-e355-492f-b3ff-c9f6e842d38b)

**Comment by engram-design** [comment]
That's an impressive formula! There's indeed a limit for the setting, roughly 60k characters, due to the `TEXT` MySQL column limit set in place for the `fields` database table where we store fields.
